test/integration/policy_generator_remote_test.go

```diff
@@ -0,0 +1,197 @@
+// Copyright Contributors to the Open Cluster Management project
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/stolostron/governance-policy-framework/test/common"
+)
+
+var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the Policy Generator "+
+	"with a remote Kustomize directory", Ordered, Label("BVT"), func() {
+	const policyName = "e2e-grc-remote-policy-app"
+	const namespace = "grc-e2e-remote-policy-generator"
+	const secret = "grc-e2e-subscription-admin-user"
+	const subAdminBinding = "open-cluster-management:subscription-admin"
+	ocpUser := common.OCPUser{
+		ClusterRoles: []types.NamespacedName{
+			{Name: "open-cluster-management:admin:local-cluster"},
+			{
+				Name:      "admin",
+				Namespace: namespace,
+			},
+		},
+		// To be considered a subscription-admin you must be part of this cluster role binding.
+		// Having the proper role in another cluster role binding does not work.
+		ClusterRoleBindings: []string{subAdminBinding},
+		Password:            "",
+		Username:            "grc-e2e-subscription-admin",
+	}
+
+	It("Sets up the application subscription", func() {
+		By("Verifying that the subscription-admin ClusterRoleBinding exists")
```

Comment:
Could you move this to a helper function rather than duplicate this?

Reply:
Fo' shizzle